ce/amf/installer.ts

```diff
@@ -46,21 +47,37 @@ export class Installs extends EntitySequence<Installer> {
     }
     throw new Error('Unsupported node type');
   }
-}
 
+  override *validate(): Iterable<ValidationError> {
```

Comment:
What do all of these `*` do?

Reply:
Marks the function as a generator (ie, something that uses `yield` instead of `return`)